skyvern/forge/sdk/workflow/models/block.py

```diff
@@ -233,47 +249,75 @@ def get_all_parameters(
 
         return parameters
 
-    def format_potential_template_parameters(self, workflow_run_context: WorkflowRunContext) -> None:
-        super().format_potential_template_parameters(workflow_run_context=workflow_run_context)
+    def format_potential_template_parameters(
+        self,
+        workflow_run_context: WorkflowRunContext,
+        extra_context: dict[str, Any] | None = None,
```

Comment:
Why not put this inside `WorkflowRunContext`?

Reply:
WorkflowRunContext is for global context and this is local context explicitly for this execution